src/client/src/routes/OrderTicketRoute/Notification.tsx

```diff
@@ -107,7 +108,7 @@ Position.defaultProps = {
   position: 'bottom',
 }
 
-const Transistion: Styled<{ visible?: boolean; position?: 'top' | 'bottom' }> & { [k: string]: any } = styled(Block)`
+const Transistion: any = styled(Block)<{ visible?: boolean; position?: 'top' | 'bottom' }>`
```

Comment:
Based on @bhavesh-desai-scratch :  we should Type every variable and avoid using any type

Reply:
Unfortunately typescript has an issue with using the `Component.from` syntax so I had to use any here. I think this component should just be rewritten in the future to use `keyframes`.